### PR TITLE
MET-19588: Update rate/delta operators docs to reflect new features

### PR DESCRIPTION
**Pull request by pkarwacki**
## Purpose of this pull request

We are introducing new features to `rate` and `delta` metrics operators. I've updated the documentation to cover these new features and also made these two pages slightly more consistent.

**IMPORTANT**
The features are not enabled for our customers yet, so we need to wait with merging these changes until then.

Issue number: MET-19588

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/metrics/metrics-operators/rate.md [hunk 1]
```diff
@@ -5,22 +5,46 @@ sidebar_label: rate
...
+The rate operator calculates the per-second rate of change between data points in the time series. Regular `rate` (without `over` modifier) calculates the rate of change between consecutive data points.
+
+`rate over TIME` on the other hand allows you to calculate an average rate of change over a specified time window. This is achieved by considering the first and the last metric value in the given time window, extrapolating to the ends of window if necessary.
 
 The rate operator also assigns the value of the metric tag to be `rate($metric)` and the value of the unit metadata field to be `$unit/second` (for example, 1/second).
 
+You can use the `increasing` or `decreasing` option, to make `rate` consider only positive or negative rate values. This functionality is useful when you are calculating the positive or the negative rate of change of a metric over time.
```
Comment (michaljmatusiak92) [inline docs/metrics/metrics-operators/rate.md:14]: minor:
```
You can use the `increasing` or `decreasing` option, to make `rate` consider only positive or negative rate values respectively.
```

File: docs/metrics/metrics-operators/delta.md [hunk 2]
```diff
@@ -9,12 +9,14 @@ The `delta` operator computes the backward difference at each data point in the
 
 The `delta` operator updates the `metric` dimension, if present, to `delta($metric)`. If the original time series does not have a `metric` dimension, it creates `metric=delta` dimension. Other dimensions remain unaffected.
 
-You can use the `increasing` or `decreasing` option, to make `delta` consider only pairs of consecutive points where the second point is greater or less than the first point. This functionality is useful when you are calculating the positive or the negative difference of a counter over time.
+You can use the `increasing` or `decreasing` option, to make `delta` consider only pairs of consecutive points where the second point is greater or less than the first point. This functionality is useful when you are calculating the positive or the negative difference of a metric over time.
```
Comment (michaljmatusiak92) [inline docs/metrics/metrics-operators/delta.md:12]: minor:
```
You can use the `increasing` or `decreasing` option, to make `delta` consider only pairs of consecutive points where the second point is greater (`increasing` option) or less (`decreasing` option) than the first point.
```

File: docs/metrics/metrics-operators/rate.md (hunk 1 above)
Comment (kimsauce) [inline docs/metrics/metrics-operators/rate.md:14]:
```suggestion
You can use the `increasing` or `decreasing` option to make `rate` consider only positive or negative rate values, respectively. This functionality is useful when you are calculating the positive or the negative rate of change of a metric over time.
```

File: docs/metrics/metrics-operators/delta.md (hunk 2 above)
Comment (kimsauce) [inline docs/metrics/metrics-operators/delta.md:12]:
```suggestion
You can use the `increasing` or `decreasing` option to make `delta` consider only pairs of consecutive points where the second point is greater (`increasing` option) or less (`decreasing` option) than the first point. This functionality is useful when you are calculating the positive or the negative difference of a metric over time.
```

**Review by kimsauce — APPROVED** [review]

**Comment by pkarwacki** [comment]
This shouldn't have been merged yet - the feature is not enabled (as was stated in the PR description). Reverting it for now.

**Pull request merged**